dataprofiler/profilers/utils.py

```diff
@@ -88,7 +89,15 @@ def shuffle_in_chunks(data_length, chunk_size):
         return []
     
     rng = np.random.default_rng()
-    if 'DATAPROFILER_SEED' in os.environ:
+
+    if settings._seed is not None:
+        try:
+            seed_value = int(settings._seed)
+            rng = np.random.default_rng(seed_value)
```

Comment:
in this case, probably don't need to set an intermediate variable.

Reply:
`seed_value`

Reply:
also, we should assume that settings._seed is in its correct form. either `None` or int > 0